arm-insights/2015-07-01/swagger/insightsManagementClient_DiagnosticsSettings.json

```diff
@@ -59,7 +59,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/ServiceDiagnosticSettingsCreateOrUpdateParameters"
+              "$ref": "#/definitions/ServiceDiagnosticSettingsResource"
             },
             "description": "Parameters supplied to the operation."
           },
```

Comment:
The change of the body is not done in the server. I need to file a bug for that to be changed before I change the spec.
ServiceDiagnosticSettings has no required property, but there must be at least one provided. Do not know how to specify that.
I just added an example of workspaceid.
The retention policies are not required.